action.yml

```diff
@@ -270,7 +271,7 @@ runs:
           "state": "${{ steps.final_state.outputs.FINAL_STATE }}",
           "context": "Testing Farm - ${{ inputs.pull_request_status_name }}",
           "description": "Build finished${{ steps.final_state.outputs.INFRA_STATE }}",
-          "target_url": "${{ env.RESULTS_URL }}"
+          "target_url": "${{ steps.sched_test.outputs.results_url }}"
```

Comment:
The changes are fine but what will be URL in the case of public ranch and in the case of private ranch?
I mean, `artifacts.dev.testing-farm.io` for public ranch and `osci....` for private ranch?

Are you sure with this change?